wifi/sender/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+find_package(PkgConfig)
+
+pkg_check_modules(GPS REQUIRED libgps)
```

Comment:
Please add the package dependencies to install to the README.md file (the top level one). This seemed to work for me:
sudo apt-get install libgps-dev libnl-genl-3-dev